desktop/src/main/java/bisq/desktop/main/overlays/windows/TradeDetailsWindow.java

```diff
@@ -358,6 +360,9 @@ private void addContent() {
                     data += "\n\nRaw deposit transaction as hex:\n" + depositTxAsHex;
                 }
 
+                data += "\n\nSelected mediator: " + DisputeAgentLookupMap.getKeyBaseUserName(contract.getMediatorNodeAddress().getFullAddress());
+                data += "\nSelected arbitrator (refund agent): " + DisputeAgentLookupMap.getKeyBaseUserName(contract.getRefundAgentNodeAddress().getFullAddress());
+
```

Comment:
Some strings are prefixed with `\n\n`, others with `\n`.  If this is the intent I'm not suggesting a change.

Reply:
Yes, its intentional. Its an additional linebreak before the first line.